spec/unit/facter/systemd_spec.rb

```diff
@@ -12,7 +12,7 @@
       end
       let(:facts) { {:kernel => :linux} }
       it do
-        Facter::Util::Resolution.expects(:exec).with('ps -p 1 -o comm=').returns('systemd')
```

Comment:
To make the current test run you could use:
```ruby
Facter::Util::Resolution.expects(:exec).with('/proc/1/exe --version').returns('systemd 233')
```

Modify ```systemd 233``` for other platforms in the other tests.